src/sections/Resume/Resume.jsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from "react";
+import { styles } from "../../styles";
+import {
+  SectionTitle,
+  SectionSubtitle,
+  TimelineMaker,
+  SkillMeter
+} from "./components";
+import { makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles(theme => ({
```

Comment:
Please create a trello card to move all styles to a separate file. I realized we are taking too much space to declare this in the component file

Reply:
Got it!